editoast/editoast_core/src/conflict_detection.rs

```diff
@@ -43,7 +43,7 @@ pub struct WorkSchedulesRequest {
 }
 
 #[derive(Debug, Deserialize, ToSchema)]
-#[cfg_attr(test, derive(Serialize))]
+#[cfg_attr(feature = "testing", derive(Serialize))]
```

Comment:
```suggestion
#[derive(Serialize)]
```

most definitions in core do not care about excessive `derive` (wrongfully though), but let's use consistency as a pretext to simplify things here...

Reply:
Done.